package.json

```diff
@@ -22,22 +22,32 @@
   "bugs": {
     "url": "https://github.com/cerner/terra-toolkit/issues"
   },
-  "browserslist": [
-    "extends browserslist-config-terra"
-  ],
   "eslintConfig": {
-    "extends": "terra",
+    "extends": "./packages/eslint-config-terra/eslint.config.js",
     "settings": {
       "polyfills": [
         "Promise"
       ]
     }
   },
   "scripts": {
+    "bootstrap:hoist": "lerna bootstrap --hoist",
     "clean": "lerna clean --yes && rm -rf node_modules",
-    "clean:install": "npm run clean && npm install"
+    "clean:install": "npm run clean && npm install",
+    "lint": "npm run lint:js",
+    "lint:js": "eslint --ext .js,.jsx .",
+    "postinstall": "npm run bootstrap:hoist"
   },
   "devDependencies": {
+    "@babel/cli": "^7.5.0",
```

Comment:
Do we need babel? Since everything in this repo will be tools, do we need to transpile any of it?

Reply:
i can remove them until we need them

Reply:
https://github.com/cerner/terra-toolkit/pull/402/commits/81534d2ce1c15eac348e43573e0b42148d28f719